### ISSUE TITLE
quesry myself

### ISSUE THREAD

**Issue by tanyabrokhman** [TechnionYP5777/Info-Evaluation#311]
Perhaps I misunderstood the idea of the project but I tried to insert the following query:
Query: Work
Name of person: Tanya Brokhman
and I get an input error "the name you're entering might have a spelling error"

same for Madonna, Michael Jackson and even for your example of :perform, Shakira"

**Comment by SlasherTech** [comment]
Hi Tanya ! :)
This is an issue of how we put down what we wanted to notify the user. (we meant to say that either the person does not exist in wikipedia OR there is a mistake in spelling).
We already got feedback for this issue and we fixed it. Tomorrow before the second presentation we will provide the users with a new APK file with this fix (I must stress that the fix was done during the presentation time and not afterwards).

As for the other problems you mentioned (trying our own examples) - I would assume you are were not connected to a Technion network while checking our project ? If so, that is the problem.
Our project uses a Technion provided server which is accessible only within the Technion.

Thank you for your feedback !
